networking/networking_operators/aws_load_balancer_operator/configuring-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-aws-load-balancer-operator"]
+= Configuring the AWS Load Balancer Operator
+include::_attributes/common-attributes.adoc[]
+:context: aws-load-balancer-operator
+
+toc::[]
+
+You can configure the cluster-wide proxy in the AWS Load Balancer Operator. After configuring the cluster-wide proxy, Operator Lifecycle Manager (OLM) automatically updates all the deployments of the Operators with the environment variables such as `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY`. These variables are populated to the managed controller by the AWS Load Balancer Operator.
+
+include::modules/configuring-egress-proxy.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
```

Comment:
Add ID here (per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies)):
```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
```

Reply:
ty